shared/src/dto/mockExamQuestion/mockExamQuestion.dto.ts

```diff
@@ -0,0 +1,32 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsArray, IsOptional, IsString, ValidateNested } from 'class-validator';
+import { DtoMockExamQuestionAnswer } from '../mockExamQuestionAnswer/mockExamQuestionAnswer.dto';
+import { DtoMockExamQuestionType } from '../mockExamQuestionType/mockExamQuestionType.dto';
+
+export class MockExamQuestionDto {
+  @ApiProperty()
+  @IsString()
+  question: string;
+
+  @ApiProperty({ nullable: true })
```

Comment:
Why don't we help out swagger with determining the value of this field? If you use a union (ie. string | undefined) it doesn't exactly know to do and defaults to `{}`. If you add @ApiProperty({ type: 'string', nullable: true }) it'll show it properly in the swagger playground